### PR TITLE
docs: canary installation

### PR DESCRIPTION
**Pull request by patzick**
## Changes

Installation of canary version

image

### Checklist

- [x] I followed [contributing](https://github.com/DivanteLtd/shopware-pwa/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ PS: Check [StorefrontUI](https://github.com/DivanteLtd/storefront-ui/) - our UI
   </tbody>
 </table>
 
+See [the documentation](https://shopware-pwa-docs.netlify.com/#introduction-to-shopware-pwa) to get started.
```
Comment (Michal-Dziedzinski) [inline README.md:33]: Better place :)

File: README.md [hunk 2]
```diff
@@ -52,6 +50,14 @@ See [the documentation](https://shopware-pwa-docs.netlify.com/#introduction-to-s
 npm install -g @shopware-pwa/cli
 ```
 
+::: tip Canary version  
+Currently, we're releasing a `canary` version per every push to `master` branch, so in order to have newest changes and fixes just install it like this:
```
Comment (Michal-Dziedzinski) [inline README.md:54]: Maybe we should mention that this version may contain some issues?

Comment (patzick) [reply README.md:54]: I thought about it, but in some cases, it's more stable than alpha release ;D
When we move to Beta stage we'll add that kind of warning, as safer then just use Beta

**Comment by patzick** [comment]
closes #581

**Pull request merged**